packages/web-app-admin-settings/src/components/Users/ContextActions.vue

```diff
@@ -66,17 +65,31 @@ export default defineComponent({
       },
       { deep: true, immediate: true }
     )
+
+    const { actions: showDetailsActions } = useActionsShowDetails()
+    const {
+      actions: editQuotaActions,
+      modalOpen: quotaModalIsOpen,
+      closeModal: closeQuotaModal,
+      spaceQuotaUpdated
+    } = useSpaceActionsEditQuota({ store })
+    const { actions: userEditActions } = useUserActionsEdit()
+    const { actions: userDeleteActions } = useUserActionsDelete({ store })
+
     const menuItemsPrimaryActions = computed(() =>
-      [...instance.$_edit_items, ...instance.$_delete_items].filter((item) =>
+      [...unref(userEditActions), ...unref(userDeleteActions)].filter((item) =>
         item.isEnabled(unref(filterParams))
       )
     )
     const menuItemsSecondaryActions = computed(() =>
-      [...instance.$_editQuota_items].filter((item) => item.isEnabled(unref(filterParams)))
+      [...unref(editQuotaActions)].filter((item) =>
+        // TODO: create separate useUserActionsEditQuota
```

Comment:
is this a future todo?

Reply:
that was part of another PR this PR was based on